src/patternfly/components/Popover/styles.scss

```diff
@@ -0,0 +1,280 @@
+//     justify-content: center;
+//     transform: translateX(-50%);
+//   }
+
+//   &.pf-m-top &__arrow {
+//     bottom: calc(-1 * var(--pf-c-popover__arrow--Size));
+//     left: calc(50% - var(--pf-c-popover__arrow--Size));
+//     border-width: var(--pf-c-popover__arrow--Size) var(--pf-c-popover__arrow--Size) 0 var(--pf-c-popover__arrow--Size);
+//     border-right-color: transparent;
+//     border-bottom-color: transparent;
+//     border-left-color: transparent;
+//   }
+
+//   &.pf-m-bottom &__container {
+//     top: calc(100% + var(--pf-c-popover__arrow--Size) + 10px);
+//     right: 50%;
+//     justify-content: center;
+//     transform: translateX(50%);
+//   }
+
+//   &.pf-m-bottom &__arrow {
+//     top: calc(-1 * var(--pf-c-popover__arrow--Size));
+//     left: calc(50% - var(--pf-c-popover__arrow--Size));
+//     border-width: 0 var(--pf-c-popover__arrow--Size) var(--pf-c-popover__arrow--Size) var(--pf-c-popover__arrow--Size);
+//     border-top-color: transparent;
+//     border-right-color: transparent;
+//     border-left-color: transparent;
+//   }
+
+//   &.pf-m-right &__container {
+//     bottom: 50%;
+//     left: calc(100% + var(--pf-c-popover__arrow--Size) + 10px);
+//     justify-content: flex-start;
+//     transform: translateY(50%);
+//   }
+
+//   &.pf-m-right &__arrow {
+//     bottom: calc(50% - var(--pf-c-popover__arrow--Size));
+//     left: calc(-1 * var(--pf-c-popover__arrow--Size));
+//     border-width: var(--pf-c-popover__arrow--Size) var(--pf-c-popover__arrow--Size) var(--pf-c-popover__arrow--Size) 0;
+//     border-top-color: transparent;
+//     border-bottom-color: transparent;
+//     border-left-color: transparent;
+//   }
+
+//   &.pf-m-compressed {
+//     --pf-c-popover__arrow--Size: var(--pf-global--arrow--BorderWidth-sm);
+//     --pf-c-popover__content--PaddingTop: var(--pf-global--spacer--sm);
+//     --pf-c-popover__content--PaddingRight: var(--pf-global--spacer--sm);
+//     --pf-c-popover__content--PaddingBottom: var(--pf-global--spacer--md);
+//     --pf-c-popover__content--PaddingLeft: var(--pf-global--spacer--md);
+//   }
+
+//   &.pf-m-inverted {
+//     --pf-c-popover__arrow--DropShadow: none;
+//     --pf-c-popover__arrow--BorderColor: var(--pf-global--Color--dark-200);
+//     --pf-c-popover__content--BackgroundColor: var(--pf-global--Color--dark-200);
+
+//     .pf-c-popover__container {
+//       --pf-c-popover__content--Color: var(--pf-global--Color--light-100);
+//     }
+
+//     .pf-c-popover__header-text,
+//     .pf-c-popover__body,
+//     .pf-c-button.pf-m-action {
+//       color: var(--pf-global--Color--light-100);
+//     }
+//   }
+// }
+
+.pf-c-popover {
+  // Component variables
+  --pf-c-popover--MinWidth: 100px;
+  --pf-c-popover--MaxWidth: 300px;
+  --pf-c-popover--BoxShadow: var(--pf-global--BoxShadow);
+  --pf-c-popover--Margin: var(--pf-global--arrow--size);
+
+  // Arrow variables
+  --pf-c-popover__arrow--Width: var(--pf-global--arrow--size);
```

Comment:
If an arrow is squared, it shouldn't have separate width and height variables but rather only a size.